luxonis_train/nodes/base_node.py

```diff
@@ -323,12 +417,16 @@ def wrap(self, output: ForwardOutputT) -> Packet[Tensor]:
                 raise IncompatibleException(
                     "Default `wrap` expects a single tensor or a list of tensors."
                 )
-        return {self._task or "features": outputs}
+        try:
+            task = self.task
```

Comment:
Should we update the docstring here to say that it wraps into a Packet with task key if present otherwise uses "features" key?